source/js/main.js

```diff
@@ -535,6 +535,49 @@ let main = {
 
       injectDonateModal(donationModal, modalOptions);
     }
+
+    // Enable the "load more results" button on index pages
+    let loadMoreButton = document.querySelector(`.load-more-index-entries`);
+    if (loadMoreButton) {
+      const entries = document.querySelector(`.index-entries`);
+
+      // Get the page size from the document, which the IndexPage should
+      // have templated into its button as a data-page-size attribute.
+      const pageSize = parseInt(loadMoreButton.dataset.pageSize) || 12;
+
+      // Start at page 1, as page 0 is the same sat as the initial page set.
+      let page = 1;
+
+      const loadMoreResults = () => {
+        // Construct our API call as a relative URL:
+        let url = `./entries/?page=${page++}&page_size=${pageSize}`;
+
+        // And then fetch the results and render them into the page.
+        fetch(url)
+          .then(result => result.json())
+          .then(data => {
+            if (!data.has_next) {
+              loadMoreButton.removeEventListener(`click`, loadMoreResults);
+              loadMoreButton.parentNode.removeChild(loadMoreButton);
+            }
+            return data.entries_html;
+          })
+          .then(entries_html => {
+            const div = document.createElement(`div`);
+            div.innerHTML = entries_html;
+
+            Array.from(div.children).forEach(child =>
+              entries.appendChild(child)
+            );
+          })
```

Comment:
In reference to Kristina's [comment](https://github.com/mozilla/foundation.mozilla.org/pull/3485#issuecomment-523062389): Maybe this would be a good place to update the results counter and get the most updated [entries length](https://github.com/mozilla/foundation.mozilla.org/pull/3485/files#diff-aa22ca63badaf6d5490c9347c4e8a92eR28) (?)

Reply:
yeah, forgot to update that span =S